extensions/wrapper/wrapper-broker-api/README.md

```diff
@@ -0,0 +1,31 @@
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/sovity/edc-extensions">
+    <img src="https://raw.githubusercontent.com/sovity/edc-ui/main/src/assets/images/sovity_logo.svg" alt="Logo" width="300">
+  </a>
+
+<h3 align="center">EDC-Connector Extension:<br />API Wrapper &amp; API Client:<br />Broker Server / Broker UI API
+Specification</h3>
+
+  <p align="center">
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=bug_report.md">Report Bug</a>
+    ·
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=feature_request.md">Request Feature</a>
+  </p>
+</div>
+
+## About this component
+
+Specification of Broker Server / Broker UI API endpoints to be included in our API Client generation for convenience.
+
+These API endpoints won't be available for either the Community Edition EDCs nor our Connector-as-a-Service / Enterprise
```

Comment:
why? I didn't get the point then, why the API exists. Say instead something like "implementation is part of broker extension"